kickstarts/partials/post/repos.ks.erb

```diff
@@ -8,6 +8,8 @@ dnf config-manager --setopt=epel.exclude=*qpid-proton* --save
 dnf config-manager --enable manageiq-*-nightly
 <% end %>
 
+dnf install -y https://resources.ovirt.org/pub/yum-repo/ovirt-release44.rpm
```

Comment:
Does this need to happen before the `%post` section in order for https://github.com/ManageIQ/manageiq-rpm_build/pull/58/files#diff-a50ce4f4455f4c6ef278a7ec5b3ab2f6R29 to work?

Reply:
No, ovirt-ansible-roles and its dependencies are all coming from existing repos specified in kickstarts/partials/main/repos.ks.erb.